broadlink/climate.py

```diff
@@ -546,16 +571,17 @@ def set_partial(self,
             'swing_h': swing_h if swing_h != None else received_state['swing_h'],
             'sleep': sleep if sleep != None else received_state['sleep'],
             'display': display if display != None else received_state['display'],
-            'health': health if health != None else received_state['health'],
-            'cmnd_0d_rmask': cmnd_0d_rmask,
-            'cmnd_0e_rmask': cmnd_0e_rmask,
-            'cmnd_18': cmnd_18,
-            'checksum_lbit': 0
+            'health': health if health != None else received_state['health']
         }
 
-        if (args['mode'] == 'heating' or args['mode'] == 'fan' or args['mode'] == 'drying'):
```

Comment:
Better to use
```python
if args['mode'] in ('heating', 'fan', 'drying'):
```

Reply:
Looks better. A suggestion:
```python3
if args['mode'] in {'heating', 'fan', 'drying'}:
```

Reply:
I think that it's less pythonic and less effective comparing to tuple

Reply:
We use sets when the order of the elements doesn't matter. They perform better in these situations and make intentions more explicit.